spec/support/macros.rb

```diff
@@ -28,5 +28,15 @@ def config_override(key, value)
         Mongoid.send("#{key}=", existing)
       end
     end
+
+    def with_config_values(key, *values)
+      values.each do |value|
+        context "when #{key} is #{value}" do
+          config_override key, value
+
+          yield
```

Comment:
do you think it's worth including the value as a parameter to `yield` here, so that it can be used when the tested result is not the same in both cases?

Reply:
That could be useful, do you want me to make this change in this PR?

Reply:
It's up to you. This change can also just be made when it's needed.